index.html

```diff
@@ -5,4 +5,4 @@
     </head>
     <body>
     </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
trailing newline, yay 😄 

(just noticed too that Fork also gives some kind of ugly warning if you don't have them)